src/Tag/index.scss

```diff
@@ -1,30 +1,27 @@
 .nds-tag {
-  padding:var(--space-xs);
+  padding: var(--space-xs) var(--space-s);
   border-radius: 16px;
-  display: flex;
-  align-items: baseline;
```

Comment:
`Row` is providing the centering behavior and the gap between the label and icon